client/my-sites/plugins/controller.js

```diff
@@ -69,6 +69,7 @@ function renderSinglePlugin( context, siteUrl, isWpcomPlugin ) {
 			prevPath: lastPluginsListVisited || context.prevPath,
 			prevQuerystring: lastPluginsQuerystring,
 			sites: sites,
+			store: context.store,
```

Comment:
If I recall correctly, passing store as a prop will enable only the child component to detect store in `connect`.
Wrapping it in provider puts store in redux context and all components down the chain can detect it.
You don't have to do it manually, we have now this:
https://github.com/Automattic/wp-calypso/pull/1955